examples/01_plotting/plot_demo_more_plotting.py

```diff
@@ -150,6 +151,61 @@
     figure=plt.figure(figsize=(5, 7)),
 )
 
+# %%
+# Using plotting threshold, vmin, vmax parameters
+# -----------------------------------------------
+#
+# Using ``threshold`` value alongside with ``vmin`` and ``vmax`` parameters
+# enable us to mask certain values in the image.
+#
+# Unthresholded image:
+
+plotting.plot_stat_map(
+    stat_img,
+    display_mode="ortho",
+    cut_coords=[36, -27, 60],
+    title="display_mode='ortho', cut_coords=[36, -27, 60]",
+)
+
+# %%
+# Image thresholded at 1:
+
+plotting.plot_stat_map(
+    stat_img,
+    threshold=1,
+    display_mode="ortho",
+    cut_coords=[36, -27, 60],
+    title="display_mode='ortho', cut_coords=[36, -27, 60]",
+)
+
+# %%
+# Setting ``vmin=0``, it is possible to plot only positive regions.
```

Comment:
for this plot and the following ones, we should use sequential colormap (like "inferno") and not the default diverging one:

see https://output.circle-artifacts.com/output/job/915e385d-a225-4516-bdd9-aa8f63d19b2c/artifacts/0/dev/auto_examples/01_plotting/plot_demo_more_plotting.html#using-plotting-threshold-vmin-vmax-parameters

![image](https://github.com/user-attachments/assets/6e58fef2-2578-451c-93f9-9cdd22ea4c61)

Reply:
makes me realize that the doc string plot_stat_map falsely claims that the colormap must be symmetrical

https://output.circle-artifacts.com/output/job/915e385d-a225-4516-bdd9-aa8f63d19b2c/artifacts/0/dev/modules/generated/nilearn.plotting.plot_stat_map.html#nilearn-plotting-plot-stat-map

![image](https://github.com/user-attachments/assets/ee54d8c5-5dc4-4b10-9096-0b14a13feb62)

Reply:
> makes me realize that the doc string plot_stat_map falsely claims that the colormap must be symmetrical

I've removed the note from docstring.

Reply:
> You mean for the plots when `vmin=0` and `vmax=0` ?

yup